modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/DynamicTableCreationTest.java

```diff
@@ -92,33 +96,26 @@ void testDynamicSimpleTableCreation() {
         assertEquals(3, clusterNodes.size());
 
         // Create table on node 0.
-        clusterNodes.get(0).tables().createTable("tbl1", tbl -> tbl
-            .changeName("tbl1")
-            .changeReplicas(1)
-            .changePartitions(10)
-            .changeColumns(cols -> cols
-                .create("key", c -> c.changeName("key").changeNullable(false).changeType(t -> t.changeType("INT64")))
-                .create("val", c -> c.changeName("val").changeNullable(true).changeType(t -> t.changeType("INT32")))
-            )
-            .changeIndices(idxs -> idxs
-                .create("PK", idx -> idx
-                    .changeName("PK")
-                    .changeType("PRIMARY")
-                    .changeColNames(new String[] {"key"})
-                    .changeColumns(c -> c
-                        .create("key", t -> t.changeName("key")))
-                    .changeAffinityColumns(new String[] {"key"}))
-            ));
+        SchemaTable schTbl1 = SchemaBuilders.tableBuilder("PUBLIC", "tbl1").columns(
+            SchemaBuilders.column("key", ColumnType.INT64).asNonNull().build(),
+            SchemaBuilders.column("val", ColumnType.INT32).asNullable().build()
+        ).withPrimaryKey("key").build();
+
+        clusterNodes.get(0).tables().createTable(schTbl1.canonicalName(), tblCh ->
+            SchemaConfigurationConverter.convert(schTbl1, tblCh)
```

Comment:
SchemaConfigurationConverter is internal class.

Reply:
It is only a test, we often use internal classes there.
After this PR we will add a method which gets a SchemaTable instead of Changer.